scripts/ci/generate-playwright-comment.sh

```
@@ -0,0 +1,143 @@
+#   - GITHUB_SHA: Current commit SHA
+#   - GITHUB_REPOSITORY: GitHub repository (owner/repo) - REQUIRED for CI
+#   - PLAYWRIGHT_REPORT_URL: Optional custom URL for Playwright HTML report
+#     (defaults to https://{owner}.github.io/{repo}/playwright/ if not set)
+#
+# Reads from playwright-report directory and extracts test results
+
+set -euo pipefail
+
+# Get run ID from environment or GitHub context
+RUN_ID="${GITHUB_RUN_ID:=unknown}"
+COMMIT_SHA="${GITHUB_SHA:=unknown}"
```

Shellcheck (0.11.0)

[warning] COMMIT_SHA appears unused. Verify use (or export if used externally).

(SC2034)